test/FlapperUniV2.t.sol

```diff
@@ -132,6 +136,29 @@ contract FlapperUniV2Test is DssTest {
         end           = EndLike(ChainlogLike(LOG).getAddress("MCD_END"));
         spotter       = SpotterLike(ChainlogLike(LOG).getAddress("MCD_SPOT"));
 
+        address farm = address(new StakingRewardsMock(PAUSE_PROXY, address(0), DAI, address(new GemMock(1_000_000 ether))));
```

Comment:
For the flapper tests consider using a mock splitter that just does vat.move to the flapper, and doing its inits separately here. That way we kind of keep the splitter and farm (imports, configs) on a higher level.

Reply:
Done